lib/board.rb

```diff
@@ -0,0 +1,51 @@
+class Board
+    def initialize
+      @board =(1..9).to_a
+      @remval=(1..9).to_a
+      @winner = nil
+      @piece = ["x","o"]
+    end
+    def replace_var(position,token)
+      @remval.delete(@board[position])
+      @board[position] = token
+    end
+    def rem_nums
+      return @remval
+    end
+    def display
+      (1..9).each do |i|
+        print " | " + @board[i - 1].to_s
```

Comment:
alll display logic should be in one class

Reply:
Please clarify what you mean by this
The display function is defined inside the board class.